storage/pebble/registers.go

```diff
@@ -74,22 +89,55 @@ func (s *Registers) Store(
 		}
 	}
 
-	err := batch.Commit(pebble.Sync)
+	err = batch.Commit(pebble.Sync)
 	if err != nil {
 		return fmt.Errorf("failed to commit batch: %w", err)
 	}
 
 	return nil
 }
 
-// TODO: Finish implementation after deciding prefixes and/or badger use
-
 // LatestHeight Gets the latest height of complete registers available
 func (s *Registers) LatestHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(latestHeightKey())
 }
 
 // FirstHeight first indexed height found in the store, typically root block for the spork
 func (s *Registers) FirstHeight() (uint64, error) {
-	panic("not implemented")
+	return s.heightLookup(firstHeightKey())
+}
+
+func (s *Registers) heightLookup(key []byte) (uint64, error) {
+	res, closer, err := s.db.Get(key)
+	defer closer.Close()
+	if err != nil {
+		return 0, err
+	}
+	return binary.BigEndian.Uint64(res), nil
+}
+
+func (s *Registers) SetFirstHeight(height uint64) error {
+	return s.indexHeightWithKey(firstHeightKey(), height)
+}
+
+func (s *Registers) SetLatestHeight(height uint64) error {
+	return s.indexHeightWithKey(latestHeightKey(), height)
+}
```

Comment:
If we plan to use these two with every read and store we should probably cache them. Can be another PR but maybe add a todo.